src/main/java/emissary/place/UnixCommandPlace.java

```diff
@@ -171,7 +171,7 @@ protected void logMessages(String tempDir) {
     public byte[] fileProcess(String[] cmd, String outputFile) {
         logger.debug("fileProcess({})", Arrays.asList(cmd));
         StringBuilder errbuf = new StringBuilder();
-        int result = executrix.execute(cmd, null, errbuf);
+        int result = executrix.execute(cmd, (StringBuilder) null, errbuf);
```

Comment:
Any particular reason for the casting of null to a StringBuilder?

In particular I'm wondering other places or Places may need this change and I want to check this doesn't break compatibility anywhere.

Reply:
There are too many overloaded methods now, it wasn't sure which one this was trying to use.